torch/onnx/symbolic_opset12.py

```diff
@@ -71,3 +71,18 @@ def nll_loss(g, self, target, weight, reduction, ignore_index):
 
 def nll_loss2d(g, self, target, weight, reduction, ignore_index):
     return nll_loss(g, self, target, weight, reduction, ignore_index)
+
+def broadcast_tensors(g, tensor_list):
+    tensors = sym_help._unpack_list(tensor_list)
+    if (tensors[0].type().dim() != 2 and tensors[0].type().sizes() != tensors[1].type().sizes()):
+        return _unimplemented("broadcast_tensors", "broadcast for tensors is only implemented in case when there are 2 tensors of the same size")
```

Comment:
_unimplemented() will throws an error like : "ONNX export failed on broadcast_tensors because broadcast for tensors is only implemented in case when there are 2 tensors of the same size not supported" which is not grammatically correct.
https://github.com/pytorch/pytorch/blob/afa8cbf8c21a080f43acfdf3cac6a07a4c606841/torch/onnx/symbolic_helper.py#L175